targets/simple_switch/simple_switch.cpp

```diff
@@ -378,12 +378,17 @@ SimpleSwitch::ingress_thread() {
             packet->save_buffer_state();
         packet->restore_buffer_state(packet_in_state);
         p4object_id_t field_list_id = clone_spec >> 16;
-        auto packet_copy = copy_ingress_pkt(
```

Comment:
The comment attached to the `copy_ingress_pkt` definition is not valid any more with this change.
Maybe you should remove `copy_ingress_pkt` altogether now that it is only used for resubmit, or try to find a smaller common denominator for ingress cloning & resubmit to avoid duplicating code.

Reply:
Thanks for the comment. Found a smaller common denominator.